tests/integration/tests/install_ibm_installation_manager_from_extracted_source.rb

```diff
@@ -0,0 +1,48 @@
+require 'im_helper'
+require 'master_manipulator'
+test_name 'FM-3804 - C94724 - Install Installation Manager from extracted source'
+
+extracted_source = '/tmp'
+source_link = nil
+
+# Teardown
+teardown do
+  confine_block(:except, :roles => %w{master dashboard database}) do
+    agents.each do |agent|
+      on(agent, "/var/ibm/InstallationManager/uninstall/uninstallc",
+         :acceptable_exit_codes => [0,127]) do |result|
+        assert_no_match(/Error/, result.stderr, 'Failed to uninstall IBM Installation Manager')
+      end
+      on(agent, "rm -rf /opt/IBM")
+      on(agent, "rm -rf #{extracted_source}/*")
```

Comment:
Good idea, this test to install and uninstall IM, so the uninstall is very often, will update in the next push